language/llama3.1-8b/download_cnndm.py

```diff
@@ -0,0 +1,127 @@
+# experiment config
+import sys
+from argparse import ArgumentParser
+import simplejson as json
+import os
+import numpy as np
+from transformers import AutoTokenizer
+from datasets import load_dataset, concatenate_datasets
+
+
+def get_args():
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--model-id",
+        type=str,
+        default="meta-llama/Meta-Llama-3.1-8B-Instruct",
+        help="Model ID to get the tokenizer",
+    )
+    parser.add_argument(
+        "--dataset-id",
+        type=str,
+        default="cnn_dailymail",
+        help="Model ID to get the tokenizer",
+    )
+    parser.add_argument(
+        "--dataset-config",
+        type=str,
+        default="3.0.0",
+        help="Model ID to get the tokenizer",
+    )
+    parser.add_argument(
+        "--text-column",
+        type=str,
+        default="article",
+        help="Model ID to get the tokenizer",
+    )
+    parser.add_argument(
+        "--summary-column",
+        type=str,
+        default="highlights",
+        help="Model ID to get the tokenizer",
+    )
+    parser.add_argument(
+        "--n-samples",
+        type=int,
+        default=None,
+        help="Model ID to get the tokenizer",
+    )
```

Comment:
I’m not certain if this pull request is ready for review, but I noticed that all the `help` messages for the command-line options are identical and appear to have been copied from the `--model-id` option. Just wanted to bring this to your attention.

Reply:
Done